### ISSUE TITLE
rename datasets

### ISSUE THREAD

**Issue by swlenz** [Rappsilber-Laboratory/xiView_container#30]
Hi, 

is there any way to implement renaming of uploaded data at a later point?

Swantje

**Comment by colin-combe** [comment]
Renaming the datasets or renaming the proteins within datasets?

**Comment by swlenz** [comment]
renaming the datasets

**Comment by colin-combe** [comment]
No, this isn't possible

**Comment by swlenz** [comment]
Could you keep it as a feature request for later?

**Comment by colin-combe** [comment]
ok

**Comment by swlenz** [comment]
Yes, I am talking about the search name. I don't know how relevant this is in our internal xi interface (but I guess if it is easy that would be nice too). 
I was thinking about it for xiVIEW, because right now it is the name of the csv file I upload. This would mean I would have to rename my csv file before to change the name.

**Comment by martingraham** [comment]
Was thinking a better way is to allow naming of a dataset when uploading, so it doesn't automatically take the filename (that would be a fallback). Wouldn't fix the problem for old searches, but would stop it being an ongoing issue.

**Comment by swlenz** [comment]
This sounds good to me
